iron-location.html

```diff
@@ -135,34 +139,54 @@
           value: false
         }
       },
+
       hostAttributes: {
         hidden: true
       },
+
       observers: [
         '_updateUrl(path, query, hash)'
       ],
+
       attached: function() {
+        this.initialize();
+      },
+
+      ready: function() {
+        if (!Polymer.version || Polymer.version >= '2') {
```

Comment:
Ugh, this check is relying on `Polymer.version` being converted to a number and then checked with another string which is converted to a number as well..Can we make it more explicit? e.g. 
```js
if (parseInt(Polymer.version[0]) >= 2)
```